src/GraphQL/Wrappers/Response/ViolationResponse.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\graphql\GraphQL\Wrappers\Response;
```

Comment:
I would make the namespace shorter and just use `Drupal\graphql\GraphQL\Response;`. All the things you introduce are response related and I think we don't need the extra "Wrapper" part?